tests/unit/scripts/test_graph.py

```diff
@@ -295,11 +295,11 @@ def test_null(null_config):
         grouping=False,
         show_suicide=False
     )
-    assert get_nodes_and_edges(opts, None, 1, 2) == ([], [])
+    assert get_nodes_and_edges(opts, '', None, 1, 2) == ([], [])
 
     opts = SimpleNamespace(
         namespaces=True,
         grouping=False,
         show_suicide=False
     )
-    assert get_nodes_and_edges(opts, None, 1, 2) == ([], [])
+    assert get_nodes_and_edges(opts, '', None, 1, 2)  == ([], [])
```

Comment:
Is `''` supposed to be the `flow_file` arg? However in `def get_nodes_and_edges()` you've put the `flow_file` arg at the end